Source/discord/discord.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#ifdef DISCORD
+
+#include <cstdint>
+#include <string>
+
+#include "../gendung.h"
+#include "../player.h"
+
+namespace discord {
+class Core;
+} // namespace discord
+
+namespace devilution {
+
+class DiscordManager {
+public:
+	DiscordManager();

```

Comment:
I'm assuming this is meant to be a singleton class, if so should probably make the constructor private. That said, does it need to be defined as a class? Only `StartGame`, `UpdateGame`, and `UpdateMenu` are used by calling code, they don't need to know anything else about the implementation. Could move the implementation inside discord.cpp with non-extern globals. random.hpp/cpp is a partial example of this approach.

Reply:
Yeah that's fair, it's left over from back when I was using it as a class in `diablo.cpp` over the singleton.